app/views/deals/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<section class="container">
+
+    <% @group_deals.each do |group_deal| %>
+        <div class="border border-1 d-flex my-3">
+          <img src="https://picsum.photos/200" width="80px">
+          <h3 
+            class="m-2"><%= group_deal.deal.name %> 
+            <span class="d-block fs-6 fw-lighter mt-3"><%= "#{group_deal.created_at.day}/#{group_deal.created_at.month}/#{group_deal.created_at.year}" %></span>
+          </h3>
+          <div class="ms-auto pt-5 pe-3">
+            <%= group_deal.deal.amount %> $
+          </div>
+        </div>
+    <% end %>
+  
+    <div class="fixed-bottom mb-3 text-center">
+     <%= link_to "New Deal", new_group_deal_path, class:"btn btn-success fs-3 w-75"  %>
+    </div>
+  
+  </section>
+  
+  
```

Comment:
- [ ] When I am on the transaction list page, there should be an arrow button ("<") that takes me to the home page. Kindly add this button and ensure it works as expected.

  ![image](https://github.com/ahmedeid6842/Budget_App/assets/36542986/87900fef-791d-4b54-aa84-240e39b1ce6d)

Reply:
Fixed